pom.xml

```diff
@@ -47,6 +54,33 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.17</version>
         </dependency>
+        <dependency>
```

Comment:
I suppose we aint this dependencies in this h\w?

Reply:
Yes, we don't need it, but I've tested examples with jsp and servlets and included these dependencies

Reply:
we can remove them